Pod/Classes/DottedLineComponent/ARDottedLineManager.m

```diff
@@ -0,0 +1,60 @@
+#import "ARDottedLineManager.h"
+
+
+
+@interface ARDottedLine : UIView
+@end
+
+
+@implementation ARDottedLine
+
+- (void)drawRect:(CGRect)rect
+{
+    [super drawRect:rect];
+    
+    [[UIColor whiteColor] setFill];
+    [[UIColor grayColor] setStroke];
+    
+    CGContextFillRect(UIGraphicsGetCurrentContext(), rect);
+    //
+    
+    const CGFloat dotDiameter = rect.size.height / 2;
+    const CGFloat gapSize = dotDiameter * 5;
+    CGFloat pattern[4];
+    pattern[0] = 0.0;
+    pattern[1] = gapSize;
+//    const CGFloat pattern = 0.5;
+    
+    UIBezierPath *path = [UIBezierPath bezierPath];
+    path.lineWidth = dotDiameter;
+    path.lineCapStyle = kCGLineCapRound;
+    [path moveToPoint:CGPointMake(0, dotDiameter / 2)];
+    [path addLineToPoint:CGPointMake(rect.size.width, 2)];
```

Comment:
Reading this code it looks like you can control the size of the dots by changing the height of the `DottedLine` component. If correct, can you add a second story that demonstrates this?